app/jobs/search_keyword_job.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class SearchKeywordJob < ApplicationJob
+  queue_as :default
+
+  def perform(keyword_id)
+    keyword = Keyword.find(keyword_id)
+
+    begin
+      GoogleSearchService.new(keyword: keyword.keyword).call
```

Comment:
As far as i check, we don't have the `keyword: ` in the `initialize` of the `GoogleSearchService`

```suggestion
      GoogleSearchService.new(keyword.keyword).call
```

Reply:
Ah yeah your right, updated here https://github.com/aetheryna/google-scraper-ruby/pull/36/commits/5ccdd75db2c408cb66f542abf903918ab83b1750